Source/WebCore/html/HTMLFormElement.cpp

```diff
@@ -356,13 +356,13 @@ RefPtr<HTMLFormControlElement> HTMLFormElement::findSubmitButton(HTMLFormControl
         return nullptr;
     RefPtr<HTMLFormControlElement> firstSuccessfulSubmitButton;
     for (auto& listedElement : m_listedElements) {
-        if (!is<HTMLFormControlElement>(*listedElement))
+        auto* control = dynamicDowncast<HTMLFormControlElement>(*listedElement);
+        if (!control)
             continue;
```

Comment:
Ditto, I have a slight preference with wrapping the whole block with `if (auto* control = dynamicDowncast<HTMLFormControlElement>(*listedElement))` instead of early `continue`.

Reply:
Same here, the body is too large, I'd rather reduce nesting.